web-extension/backend/admin-ns.js

```diff
@@ -23,9 +26,36 @@ function doValidate(validator, params) {
  * @returns {JSONRPCServer}
  */
 export default function init({ encryptedStore, settings, wallets, networks, connections, onerror }) {
+  let popout = null
+  windows.onRemoved.addListener(() => {
```

Comment:
@dexturr should we perhaps check the `windowId` here just to be absolutely sure we are setting the popout to null in the right cases? https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/windows/onRemoved#parameters

Reply:
Ah this breaks BE tests. We can fudge it a bit by moving this listener into the setup of the popup and remove it on close. I'll do that monday

Reply:
Have made the change for the windowId stuff